app/utils/stacking/pox.ts

```diff
@@ -25,20 +28,38 @@ interface POXInfo {
   reward_cycle_length: number;
 }
 
-interface StackerInfo {
-  amountSTX: BN;
-  firstRewardCycle: BN;
-  lockPeriod: BN;
+export interface StackerInfo {
+  amountSTX: string;
+  firstRewardCycle: number;
+  lockPeriod: number;
   poxAddr: {
     version: Buffer;
     hashbytes: Buffer;
   };
   btcAddress: string;
 }
 
+interface StackerInfoCV {
+  'amount-ustx': UIntCV;
+  'first-reward-cycle': UIntCV;
+  'pox-addr': {
```

Comment:
Hypen-based naming used elsewhere spreads like a highly infectious, deadly illness. Is this something we just have to live with given clarity's const naming 🤔

Reply:
lol, agreed. In this case we'll need the hyphens no matter what for this type signature. When auto-generated clarity clients with camelcase conversion??